gpflow/conditionals/conditionals.py

```diff
@@ -66,13 +66,12 @@ def _conditional(
         Please see `gpflow.conditional._expand_independent_outputs` for more information
         about the shape of the variance, depending on `full_cov` and `full_output_cov`.
     """
-    Kmm = Kuu(inducing_variable, kernel, jitter=default_jitter())  # [M, M]
-    Kmn = Kuf(inducing_variable, kernel, Xnew)  # [M, N]
-    Knn = kernel(Xnew, full_cov=full_cov)
-    fmean, fvar = base_conditional(
-        Kmn, Kmm, Knn, f, full_cov=full_cov, q_sqrt=q_sqrt, white=white
-    )  # [N, R],  [R, N, N] or [N, R]
-    return fmean, expand_independent_outputs(fvar, full_cov, full_output_cov)
+    posterior_class = get_posterior_class(kernel, inducing_variable)
+
+    posterior = posterior_class(
+        kernel, inducing_variable, f, q_sqrt, whiten=white, mean_function=None, precompute=False
+    )
+    return posterior.fused_predict_f(Xnew, full_cov=full_cov, full_output_cov=full_output_cov)
```

Comment:
Fallback to enable conditionals() to still work without users having to register both get_posterior_class() *and* conditional().

Questions:
1. should we add a deprecation warning when users try to register on conditional() directly?
2. should we just remove the explicit multi-output conditional() implementations (which now do exactly the same as this fallback)?